scripts/circleci/dependencies-functions.sh

```diff
@@ -26,24 +27,35 @@ dependencies_build_setup() {
         sudo ln --symbolic "/usr/local/groovy/bin/groovysh" /usr/local/bin/groovysh &&
         sudo ln --symbolic "/usr/local/groovy/bin/groovyc" /usr/local/bin/groovyc
 
-    mkdir -p "$HOME/.gradle"
 }
 
+# Install nodejs
+dependencies_install_nodejs() {
+  node_version=$(cat .nvmrc)
+  curl -sSl -o /tmp/node.tar.xz "https://nodejs.org/dist/${node_version}/node-${node_version}-linux-x64.tar.xz" && \
+  echo '592eb35c352c7c0c8c4b2ecf9c19d615e78de68c20b660eb74bd85f8c8395063 /tmp/node.tar.xz' | sha256sum --check && \
```

Comment:
Won't this sha change per version? Kind of strange to read from `.nvmrc`  but then have this hard-coded?

Reply:
Yes i'm undecided on this, on one side i want to base on nvmrc to not having multiple node version placeholders, from other side i'd like to have the version sha hardcoded to prevent supply chain attacks.

Moved this change to another PR: https://github.com/rundeckpro/rundeckpro/pull/3981